src/components/buoy/ExploreForm.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <BaseForm>
+    <template #inputs>
+      <div class="control-item">
+        <label for="buoy-select" class="label">Select Buoys</label>
+        <multiselect
+          id="buoy-select"
+          v-model="selectedBuoys"
+          class="multiselect"
+          :options="store.coordinates"
+          :custom-label="formatCoordinate"
+          track-by="station_name"
+          :multiple="true"
+        />
+      </div>
+
+      <div class="control-item">
+        <label for="variable-select-visualize" class="label"
+          >Select Variables (up to 4)</label
```

Comment:
Is this a suggestion? Is there a way to limit this?

<img width="834" alt="Screen Shot 2022-01-05 at 1 38 10 PM" src="https://user-images.githubusercontent.com/5270855/148270671-d2a4ed9d-9bb2-4cb5-9748-ae4f9c1dc95d.png">

Reply:
you can't hit submit if you have more than 4

Reply:
Oh I see it now! I didn't try to submit before